sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslateContent.Serialization.cs

```diff
@@ -38,7 +38,7 @@ internal virtual MultipartFormDataRequestContent ToMultipartRequestContent()
             writer.WriteStartObject();
             writer.WritePropertyName("document"u8);
 #if NET6_0_OR_GREATER
-				writer.WriteRawValue(global::System.BinaryData.FromStream(Document));
```

Comment:
This doesn't compile, based on the else statement, this looks like the correct code. Please let me know if it's not.